### DISCUSSION TITLE
No change in #/Taxa when trying tax_select?

### DISCUSSION THREAD

**Discussion by hank00000** [david-barnett/microViz#101] [Q&A]
Looking to subset all methanotrophs/methanogens by using general taxa search "meth", which is in all the taxa of interest's names (ex. Methanolobulus, Methylovirgula, etc). I saw the example used for "Bif" in this package tutorial and thought this would work, but my original and subsetted objects have the same number of taxa after using below code: 

Example code:

physeqMETH <- (tax_select(physeq, tax_list = "Meth", n_typos = 0, ranks_searched = "Family"))

physeq (original): phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 4976 taxa and 68 samples ]
sample_data() Sample Data:       [ 68 samples by 4 sample variables ]
tax_table()   Taxonomy Table:    [ 4976 taxa by 7 taxonomic ranks ]
phy_tree()    Phylogenetic Tree: [ 4976 tips and 4953 internal nodes ]

physeqMETH: phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 4976 taxa and 68 samples ]
sample_data() Sample Data:       [ 68 samples by 4 sample variables ]
tax_table()   Taxonomy Table:    [ 4976 taxa by 7 taxonomic ranks ]
phy_tree()    Phylogenetic Tree: [ 4976 tips and 4953 internal nodes ]
>

**Comment by david-barnett** [comment]
#102 might be a bug so I opened an issue, can you reply there please? :)
